zeebe/engine/src/main/java/io/camunda/zeebe/engine/state/deployment/DbResourceState.java

```diff
@@ -140,9 +141,6 @@ public void storeResourceInResourceColumnFamily(final ResourceRecord record) {
     dbResourceKey.wrapLong(record.getResourceKey());
     dbPersistedResource.wrap(record);
     resourcesByKey.upsert(tenantAwareResourceKey, dbPersistedResource);
-    resourcesByTenantIdAndIdCache.put(
-        new DbResourceState.TenantIdAndResourceId(record.getTenantId(), record.getResourceId()),
-        dbPersistedResource.copy());
```

Comment:
:thought_balloon: This could stay IMO, it avoids an unnecessary cache load for recently deployed resources.

Reply:
My reasoning was to avoid overfilling the cache, but we'll always have the latest resource versions on hand this way. 👍